src/seismometer/data/loader/pipeline.py

```diff
@@ -65,6 +66,7 @@ def __init__(
         post_predict_fn: Optional[ConfigFrameHook] = None,
         post_event_fn: Optional[ConfigFrameHook] = None,
         merge_fn: Optional[MergeFramesHook] = None,
+        post_load_fn: Optional[ConfigFrameHook] = None,
```

Comment:
Does it also make sense to do some validation on the code hook itself? Pydantic appears to have some support for this already -  https://docs.pydantic.dev/latest/api/validate_call/#pydantic.validate_call_decorator.validate_call

Reply:
Played with this a bit, and think I want to push this kind of validation assistance to the larger planned RFC around more config-based support for post_load transforms.

The validate_call is kind of the opposite direction than we want, and to help we'd have to 1-require that post_load_fn be defined with annotations and then do inspection on the signature.